SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -22,16 +22,16 @@ def get(self):
         """
         self.set_header("Content-Type", "text/event-stream")
 
-        repository = self.get_query_argument("repo", default=None)
-        repository_type = self.get_query_argument("repo_type", default=None)
-        accpy_version = self.get_query_argument("accpy", default=None)
+        repository = self.get_query_argument("repo", default='')
```

Comment:
Style comment: it is better if you stick to either single or double quotes for string literals throughout a document. Since you already have double quotes (at least in this fragment), I would use double for all the changes of this PR.